lib/mycha.coffee

```diff
@@ -77,7 +77,7 @@ class Mycha
   # Parameters:
   # - mocha_args: Array of string arguments to provide to Mocha.
   call_mocha: (mocha_args, done) ->
-    childProcess = child.spawn path.resolve(__dirname, '../node_modules/mocha/bin/mocha'),
+    childProcess = child.spawn path.join(require.resolve('mocha'), '..', 'bin', 'mocha'),
```

Comment:
This results in path.join `[...]/node_modules/mocha/index.js/../bin/mocha` which works because path.join normalizes the path and it doesn't know that index.js isn't a directory. But it still seems a bit strange.

What do you think about `require.resolve 'mocha/bin/mocha'` for now?